README.md

```diff
@@ -1,29 +1,23 @@
 PostSignal
 ==========
-
-Opens up a url endpoint to which you can POST json objects. These will then be output as signals.
+DEPRECATED, USE web_hanlder BLOCKS INSTEAD - Opens up a url endpoint to which you can POST json objects. These will then be output as signals.
 
 Properties
 ----------
-- **endpoint**(string): Endpoint at which to accept signals.
-- **host**(string): Local IP. Defaults to 127.0.0.1.
-- **include_headers**(bool): Whether to include headers on post response
-- **port**(int): Port to accept signals on. Defaults to 8182.
-- **response_headers**(object): Headers to include on post response
+- **endpoint**: Endpoint at which to accept signals.
+- **host**: Local IP. Defaults to 127.0.0.1.
+- **include_headers**: Whether to include headers on post response
+- **port**: Port to accept signals on. Defaults to 8182.
+- **response_headers**: 
```

Comment:
This description got deleted but I guess it doesn't matter too much since this is deprecated